### PR TITLE
Update language.json

### PR DESCRIPTION
**Pull request by cagatayldzz**

### CODE REVIEW

**Comment by TobiTenno** [comment]
Ki'Teer has an upper case T

File: data/languages.json [hunk 1]
```diff
@@ -10830,8 +10839,11 @@
   "/lotus/storeitems/upgrades/skins/voidtrader/baroinarospolearmskin": {
     "value": "Anpu Staff Skin"
   },
+  "/lotus/storeitems/upgrades/skins/voidtrader/elixistigris": {
+    "value": "Elixis Tigris Skin"
+  },
   "/lotus/storeitems/upgrades/skins/voidtrader/elixissonicor": {
-    "value": "Sonicor Exilis Skin"
+    "value": "Elixis Sonicor Skin"
```
Comment (TobiTenno) [inline data/languages.json]: the format i'm seeing elsewhere, especially in the mobile app, is `${item} ${skin family} Skin`, not `${skin family} ${item} Skin`... do you have a screenshot, perchance? I wasn't around all weekend to see it, sadly.

Comment (cagatayldzz) [reply data/languages.json]: edited

Comment (cagatayldzz) [reply data/languages.json]: * https://warframe.fandom.com/wiki/Sonicor_Elixis_Skin
"Sonicor Elixis Skin"

File: data/languages.json [hunk 2]
```diff
@@ -10830,8 +10839,11 @@
   "/lotus/storeitems/upgrades/skins/voidtrader/baroinarospolearmskin": {
     "value": "Anpu Staff Skin"
   },
+  "/lotus/storeitems/upgrades/skins/voidtrader/elixistigris": {
+    "value": "Elixis Tigris Skin"
```
Comment (TobiTenno) [inline data/languages.json]: this should be the other way as well

**Comment by TobiTenno** [comment]
I don't tend to trust the wiki as the single source of truth on name ordering

File: data/languages.json (hunk 2 above)
Comment (cagatayldzz) [reply data/languages.json]: this is the name that appears in the Baro Ki'Teer
not wrong

Comment (cagatayldzz) [reply data/languages.json]: image will be added for accuracy when next pull request is made

Comment (TobiTenno) [reply data/languages.json]: you can upload into comments....

**Pull request closed without merging**